### PR TITLE
Rc/3.13.1

### PR DESCRIPTION
**Pull request by samrichca**
## New Features

* _banners_ type added to gql schema, data provided through env config

### CODE REVIEW

**Review by joneubank — APPROVED** [review]

File: src/schemas/Banner/index.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  title: string;
+}
+
+const allowedFields = [
+  'dismissable',
+  'id',
+  'level',
+  'message',
+  'title',
+];
```
Comment (hlminh2000) [inline src/schemas/Banner/index.ts:31]: if you type this as `AllowedFields[]`, it'll help avoid mistakes in forgetting to add fields when this changes ;)

Comment (hlminh2000) [reply src/schemas/Banner/index.ts:31]: actually, `AllowedFields` can just be `keyof Banner` too

Comment (ciaranschutte) [reply src/schemas/Banner/index.ts:31]: god definition i like that

Comment (samrichca) [reply src/schemas/Banner/index.ts:38]: i updated `AllowedFields` to be `keyof Banner` and added a comment about the other issue

tl;dr we're doing runtime checks because there's no other data validation for the banners and we don't want the build to crash
